pandas/_libs/window.pyx

```diff
@@ -1263,12 +1263,12 @@ cdef _roll_min_max(ndarray[numeric] values, int64_t win, int64_t minp,
         return _roll_min_max_variable(values, starti, endi, N, win, minp,
                                       is_max)
     else:
-        return _roll_min_max_fixed(values, starti, endi, N, win, minp, is_max)
+        return _roll_min_max_fixed(values, N, win, minp, is_max)
 
 
 cdef _roll_min_max_variable(ndarray[numeric] values,
```

Comment:
Any reason for keeping this an ndarray versus a memory view?

Reply:
There is no need to convert that one to a memoryview.  The values array comes from a python object and doesn't have the same memory leak issue as the `ndarry`s that comes from the cdef in `_roll_min_max`